src/CommunityToolkit.Maui.MediaElement/Interfaces/IMediaElement.cs

```diff
@@ -135,6 +135,16 @@ public interface IMediaElement : IView
 	/// </summary>
 	void Stop();
 
+	/// <summary>
+	/// Sets the player to full screen.
+	/// </summary>
+	void FullScreen();
```

Comment:
Let's ensure that we are using verbs for method names, eg `EnlargeVideoToFullScreen` or something similar that demonstrates the action that the method will perform